services/frontend-service/src/ui/Warnings.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
+import type { Environment, Release } from '../api/api';
+
+export enum DeployState {
+    Normal,
+    Undeploy,
+    Mixed,
+}
+
+export function getDeployState(name: string, environments: { [name: string]: Environment }): DeployState {
+    let allNormal = true;
+    let allUndeploy = true;
+    for (const envName in environments) {
+        const application = environments[envName].applications[name];
+        if (application) {
+            if (application.undeployVersion) {
+                allNormal = false;
+            } else {
+                allUndeploy = false;
+            }
+        }
+    }
+    if (allNormal) {
+        return DeployState.Normal;
+    }
+    if (allUndeploy) {
+        return DeployState.Undeploy;
+    }
+    return DeployState.Mixed;
+}
+
+const UndeployRunningWarning: React.FC<any> = (props: { deployState: DeployState }) => {
+    const tooltip = 'This app is in the process of deletion';
+    const undeployHint = (
```

Comment:
Please use the `<Tooltip />` component for consistency.
https://github.com/freiheit-com/kuberpult/blob/b77ddec016b456d524039f17cdfd1db31fb18d86/services/frontend-service/src/ui/ReleaseDialog.tsx#L69

Reply:
done